localStorage-Value-Checker/README.md

```diff
@@ -0,0 +1,11 @@
+# localStorage Value Checker
+
```

Comment:
do you mind checking the standard README and formatting your accordingly? 

https://github.com/zendesk/demo_apps/blob/master/basic_user_sample_app/README.md